cmd/kube-scheduler/app/options/insecure_serving_test.go

```diff
@@ -42,8 +48,8 @@ func TestOptions_ApplyTo(t *testing.T) {
 			name: "no config, zero port",
 			options: Options{
 				ComponentConfig: kubeschedulerconfig.KubeSchedulerConfiguration{
-					HealthzBindAddress: "1.2.3.4:1234",
-					MetricsBindAddress: "1.2.3.4:1234",
+					HealthzBindAddress: servingAddress("1.2.3.4:1234"),
```

Comment:
You might use `k8s.io/utils/pointer.StringPtr` [1] instead.

[1] https://github.com/kubernetes/utils/blob/master/pointer/pointer.go#L74